composer.json

```diff
@@ -12,13 +12,13 @@
         "php": "^5.6 || ^7.0",
         "php-http/httplug": "^1.0 || ^2.0",
         "php-http/client-implementation": "^1.0",
+        "php-http/discovery": "^1.7",
```

Comment:
@greg0ire I had to move this back to "require". Not having it here will break the code Im afraid.

https://github.com/php-translation/translator/blob/master/src/Service/HttpTranslator.php#L41

Reply:
Good call 👍